### PR TITLE
Prepare for publishing v2.1.0-alpha2

### PR DESCRIPTION
**Pull request by jiangliu**
Prepare for publishing v2.1.0-alpha2 to fix a bug in alpha, and relax
dependency constraints.

Signed-off-by: Jiang Liu <gerry@linux.alibaba.com>

### CODE REVIEW

**Comment by hsiangkao** [comment]
can we retag 2.1.0-alpha1? since it's actually not released yet.

**Comment by hsiangkao** [comment]
can we retag 2.1.0-alpha1? since it's actually not released yet.

**Comment by jiangliu** [comment]
> can we retag 2.1.0-alpha1? since it's actually not released yet.

It has been published to crates.io, seems we can't publish the same version crates.io again.

**Review by hsiangkao — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -56,7 +56,7 @@ virtio-bindings = { version = "0.1", features = ["virtio-v5_0_0"], optional = tr
 virtio-queue = { version = "0.1.0", optional = true }
 
 nydus-api = { version = "0.1.0", path = "api" }
-nydus-app = { version = "0.2.0", path = "app" }
+nydus-app = { version = "0.3.0", path = "app" }
```
Comment (hsiangkao) [inline Cargo.toml:59]: 0.3?

File: Cargo.lock [hunk 2]
```diff
@@ -1093,7 +1093,7 @@ dependencies = [
 
 [[package]]
 name = "nydus-app"
-version = "0.2.0"
+version = "0.3.0"
```
Comment (hsiangkao) [inline Cargo.lock:1096]: 0.3?

File: blobfs/Cargo.toml [hunk 3]
```diff
@@ -17,12 +17,12 @@ serde_json = "1.0.53"
...
-nydus-error = { version = "0.2.0", path = "../error" }
-nydus-rafs = { version = "0.1.0", path = "../rafs" }
-nydus-storage = { version = "0.5.0", path = "../storage", features = ["backend-localfs"] }
+nydus-error = { version = "0.2", path = "../error" }
+nydus-rafs = { version = "0.1", path = "../rafs" }
+nydus-storage = { version = "0.5", path = "../storage", features = ["backend-localfs"] }
 
 [dev-dependencies]
-nydus-app = { version = "0.2", path = "../app" }
+nydus-app = { version = "0.3", path = "../app" }
```
Comment (hsiangkao) [inline blobfs/Cargo.toml:25]: according here.

**Review by hsiangkao — APPROVED** [review]

**Pull request merged**